### ISSUE TITLE
TF Flame Thrower = instant Death

### ISSUE THREAD

**Issue by egrain** [DarkPlacesEngine/darkplaces#17]
Team Fortress version 2.8 (2.9 doesn't work, because there is only a qwprogs.dat available)

If you run into the flames of the flamethrower you die almost instantly. We tried it even with HWGuy 200armor 100health: 1 Mississip.. dead. Flamethrower damage is like in QW when you walk away from Pyro or just stand.

Also walking down the spiral slopes in 2fort5 (and other variants) can lead to instant death when you are touching a wall while you do it. "Fell to death" message appears.

Both issues might have the same origin. I don't know though. Thanks for working on this engine again. We greatly enjoy it. Have been playing TF regularly on it for almost two years now.

**Comment by egrain** [comment]
Don't slow down the walking speed. It's the main reason why we are playing Darkplaces/TF and not Fortressone. Seriously, we rather have the trip and fall bug and the flame thrower instant death than the constant jumping around. No offense.

**Comment by Cloudwalk9** [comment]
If you can, please download the latest master, compile it per the instructions in the README, and test again.

**Comment by hemebond** [comment]
@egrain Have you tested on the latest version of DarkPlaces from source? Please also link to the TF source code and downloads if it is still an issue.

**Comment by egrain** [comment]
We are still playing 2014. I wrote my own fork of TF, and I took out the collisions of the flames, and added a falling damage cool down. So all good.
I don't trust your new releases, because I tested out sdl, since glx isn't being worked on from what I heard, and there 7.1 sound didn't work, which is very important to me. Oh, I mean in Linux. In Windows it hasn't been working for quite a while, but then who still uses that?
